roles/scalelite/defaults/main.yml

```diff
@@ -0,0 +1,207 @@
+---
+scalelite_docker_volumes_folder: /docker_volumes
+scalelite_docker_compose_folder: /scalelite
+scalelite_docker_compose_owner: root
+scalelite_docker_compose_group: root
+
+scalelite_certbot_image: certbot/certbot:v2.11.0
```

Comment:
image tag variable would be nice

Reply:
Do we still plan to use hashes here at a later point (DBP-1340)?
That could be incompatible with a traditional image_tag variable, because the syntax is a bit different:
`<image>@sha256:<hash>`

Reply:
You are right Yannick, so my suggestion would be: use only the image variable without an explicit tag variable. 
And in the image variable we can define a tag or a hash.